src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -125,7 +125,7 @@ protected function getDumpHeader()
     rxEsc = /([.*+?^${}()|\[\]\/\\])/g,
     idRx = /\bsf-dump-\d+-ref[012]\w+\b/;
 
-doc.documentElement.firstChild.appendChild(refStyle);
+doc.documentElement.firstElementChild.appendChild(refStyle);
```

Comment:
After a but of research & tests, `children[0]` should be used for IE7&8 (the component works there):
`(doc.documentElement.firstElementChild || doc.documentElement.children[0]).appendChild(refStyle);`